superset/dashboards/dao.py

```diff
@@ -100,6 +101,25 @@ def get_dashboard_and_slices_changed_on(  # pylint: disable=invalid-name
         # drop microseconds in datetime to match with last_modified header
         return max(dashboard_changed_on, slices_changed_on).replace(microsecond=0)
 
+    @staticmethod
+    def get_dashboards_created_by(user_id: int) -> List[Dashboard]:
+        """
+        Gets a list of dashboards that were created or changed by a certain user
+        :param user_id: The user id
+        :return: List of dashboards
+        """
+        qry = (
+            db.session.query(Dashboard)
+            .filter(  # pylint: disable=comparison-with-callable
+                or_(
+                    Dashboard.created_by_fk == user_id,
+                    Dashboard.changed_by_fk == user_id,
+                )
+            )
+            .order_by(Dashboard.changed_on.desc())
+        )
+        return qry.all()
```

Comment:
per the question above, yeah, this could totally work as a filter on the api, no?

Reply:
Most probably, I'll make the change